LastWork(newWork)/Views/new-evaluation/newEvaluationPage.js

```diff
@@ -316,35 +230,163 @@ const DropdownSelector = function (option) {
+    },
+};
+
+interviewApp.TechnicalLevelPickerHeader = function (option) {
+    const levelTitle = option.headers.map(element => `
+        <span class="technicalLevel_span"> ${element} </span>
+        `);
+    return levelTitle.join('');
+};
+
+interviewApp.TechnicalLevelPickerInputs = function (option) {
+    const inputsArr = option.map(element => `
+             <input type="radio" class="radioInput" name="radioInput" value = "${element}">
+            `);
+    return `
+        <div class="groupRadioButtons">
+        ${inputsArr.join('')}
+        </div>`;
+};
+
+interviewApp.TechnicalLevelPickerBody = function (option) {
+    const radioList = option.columnData.map(element => `
+        ${interviewApp.TechnicalLevelPickerInputs(element.inputLevels)}
+        `);
+    return radioList.join('');
+};
+
+interviewApp.TechnicalLevelPicker = {
+    init() {
+        const obj = interviewApp.newEvalObj.TechnicalObj;
+
+        return `
+    <h1 class="boxTitle">${obj.title}</h1>
+    <div id="radio_select">
+        <div id="radio_select_header">
+            ${interviewApp.TechnicalLevelPickerHeader(obj)}
+        </div>
+            ${interviewApp.TechnicalLevelPickerBody(obj)}
+    </div>
+    `;
+    },
+    destroy() {
+        document.querySelector('#technicalLevelPicker').innerHTML = '';
+    },
+};
+
+interviewApp.textArea = function (option) {
+    return `
+    <h1 class="boxTitle">${option.title}</h1>
+    <textarea id="${option.id}" name="${option.name}" placeholder="${option.placeholder}"></textarea>
+    `;
+};
+
+interviewApp.textAreaSection = {
+    init() {
+        const obj = interviewApp.newEvalObj.textAreaobj;
+
+        const textAreas = obj.map(element => interviewApp.textArea(element)).join('');
+        return `
+        ${textAreas}
+        `;
+    },
+};
 
-    const fieldSetListString = dropDowns.map(element => FieldSet(element)).join('');
-    return fieldSetListString;
+interviewApp.DropDownListItem = function (option) {
+    return `
+    <li>
+    <label>${option.label}</label>
+    <select id="${option.name}" name = "${option.name}">
+        <option selected disabled> Choose </option>
+        <option value="0">None</option>
+        <option value="1">Low</option>
+        <option value="2">Medium</option>
+        <option value="3">High</option>					
+    </select>
+    </li>
+    `;
 };
 
-const CandidatePage = function (option) {
+interviewApp.FieldSet = function (option) {
+    const fieldSetList = option.items.map(element => interviewApp.DropDownListItem(element)).join('');
+
     return `
+    <fieldset>
+    <legend id="${option.legend}">${option.legend}</legend>
+        <ul>
+            ${fieldSetList}
+        </ul>
+    </fieldset>
+    `;
+};
+
+interviewApp.DropdownSelector = {
+    init() {
+        const dropDowns = interviewApp.newEvalObj.dropDownsObj;
+
+        const fieldSetListString = dropDowns.map(element => interviewApp.FieldSet(element)).join('');
```

Comment:
Pedantic, really, but this constant is actually superfluous and can be rewritten to simply `return` its value directly.